src/Entity.php

```diff
@@ -3120,4 +3126,50 @@ public static function getEntitySelectorTree(): array
 
         return $entitiestree;
     }
+
+    public function showHelpdeskHomeConfig(): bool
+    {
+        $tiles_manager = new TilesManager();
+
+        if (
+            // Is not root entity
+            $this->getId() !== 0
+            // Editable
+            && static::canUpdate()
+            && $this->canUpdateItem()
+            // Has no tiles
+            && count($tiles_manager->getTilesForItem($this)) === 0
+        ) {
+            // Render a custom template when there are no tiles as we want the
+            // user to preview the tiles from the parent entities and to be able
+            // to copy them into the current entity if needed.
+            $twig = TemplateRenderer::getInstance();
+            $twig->display(
+                'pages/admin/helpdesk_home_config_for_empty_entity.html.twig',
+                [
+                    'tiles_manager' => $tiles_manager,
+                    'itemtype_item' => static::class,
+                    'items_id_item' => $this->getID(),
+                    'info_text'     => $this->getConfigInformationText(),
+                    'parent_tiles'  => $tiles_manager->getTilesForEntityRecursive($this)
+                ]
+            );
+        } else {
+            $tiles_manager->showConfigFormForItem($this);
+        }
+
+        return true;
+    }
+
+    #[Override]
+    public function acceptTiles(): bool
+    {
+        return true;
+    }
+
+    #[Override]
+    public function getConfigInformationText(): ?string
```

Comment:
Please note that this method name is not really helping in th Entity context. It should be changed, but please do it in another PR so we can merge this one and integrate it in the `11.0.0-beta` release.